docs/user-documentation/content_types.md

```diff
@@ -0,0 +1,126 @@
+1. Save. When creating a new Repository Item, the *Needs Review?* Field appears at the top, as a set of radio buttons.
+
+### Change the Content Display
+
+Finally, let’s change how the results of this field are displayed. As things stand now, our new field shows up at the bottom of repository object pages:
+
+![a screenshot of a "Needs Review?" field in the node display](/docs/assets/islandora8_fieldindisplay.png)
+
+In the Admin menu, return to Structure > Content Types and find the Repository Item content type again. Select Manage display.
+
+1. Find the *Needs Review?* We can change how the field title is display (inline/above/hidden entirely) and replace the options displayed with variations on a binary choice (yes/no, enabled/disabled, checkmark/X) or hide the field completely. 
+1. We can also drag the field into the Disabled section so that neither its label or its contents appear in the display, although the area saved on the node.
+1. Drag the field to "Disabled" and save.
+1. We no longer see the field on the display, but it is available when editing the node.
+
+## Create a Content Type
+
+To create your own custom content type from scratch, please refer to [this guide](https://www.drupal.org/docs/8/administering-drupal-8-site/managing-content-0/create-a-custom-content-type) on Drupal.org.
+
+
+## Update / Create an RDF Mapping
+
+RDF mapping is aligning drupal fields to RDF ontology properties. For example the title field of a content model can be mapped to dcterms:title and/or schema:title. In Islandora 8, triples expressed by these mappings get synced to Fedora and indexed in the Blazegraph triplestore. RDF mappings are defined/stored in Drupal as a YAML file. Currently, Drupal 8 does not have a UI to create/update RDF mappings to ontologies other than Schema.org. This requires repository managers to update the configuration files themselves. 
+
+The Drupal 8 Configuration Synchronization export (e.g. `http://localhost:8000/admin/config/development/configuration/single/export`) and import (e.g. `http://localhost:8000/admin/config/development/configuration/single/import`) can be used to get a copy of the mappings for editing in a text editor before being uploaded again. Alternatively, a repository manager can update the configuration on the server and use Features to import the edits.
+
+RDF mappings, like all Drupal configuration files, are written in [YAML](https://yaml.org/). YAML is out of scope for this documentation, but there are [several tutorials on the web](https://duckduckgo.com/?q=yaml+tutorial). 
```

Comment:
This line is now redundant with the opening paragraph, although the YAML link should be incorporated above.